documentation/book/api/paths.adoc

```diff
@@ -770,6 +770,73 @@ __required__|List of topics to which the consumer will subscribe.|<<_topics,Topi
 ----
 
 
+[[_listsubscriptions]]
+=== GET /consumers/{groupid}/instances/{name}/subscription
+
+==== Description
+Retrieves a list of the topics to which is the consumer subscribed.
```

Comment:
```suggestion
Retrieves a list of topics that a consumer is subscribed to. The request for this operation must use the base URL (including the host and port) returned in the response from the `POST` request to `/consumers/{groupid}` that was used to create this consumer.
```

Reply:
This file is generated. It takes the text from openapi file.

Reply:
Please can you update it in the definition file?